packages/shared/ReactFeatureFlags.js

```diff
@@ -123,3 +123,6 @@ export const enableLegacyFBSupport = false;
 // interleaved event. Remove this flag once we have migrated to the
 // new behavior.
 export const deferRenderPhaseUpdateToNextBatch = true;
+
+// Replacement for runWithPriority in React internals.
+export const enableCurrentUpdateLanePriority = false;
```

Comment:
Let's name this something higher level, like `decoupleUpdatePriorityFromScheduler`.